docs/api_auth.md

```diff
@@ -0,0 +1,32 @@
+# Relay API Authentication
+The Relay API is built on [Django REST Framework][drf] and authenticates
+requests with any of 3 methods:
+* [FXA OAuth Token Authentication](fxa-oauth-token-authentication)
+* [`SessionAuthentication`][SessionAuthentication]
+* [`TokenAuthentication`][TokenAuthentication]
+
+## FXA OAuth Token Authentication
+The Relay add-on uses the [`identity.launchWebAuthFlow` API][mdn-webauthflow]
+to perform an OAuth2 flow with [the FXA OAuth service][fxa-oauth], including
+[PKCE][fxa-pkce].
+
+After the OAuth flow is complete, the add-on has an FXA access token and a
+long-living FXA refresh token, and authenticates all requests to the Relay
+server by including an `Authorization: Bearer {fxa-access-token}` header in all
+API requests. The Relay server checks the token against [the FXA OAuth `/verify`
+endpoint][fxa-oauth-token-verify].
```

Comment:
This embeds the newline in the link text - add an early line break to avoid it?
```suggestion
API requests. The Relay server checks the token against
[the FXA OAuth `/verify` endpoint][fxa-oauth-token-verify].
```